src/style.css

```
@@ -1,3 +1,70 @@
-body {
+* {
```

add css variable which will store color and reuse it.
```suggestion
:root {
  --main-color: #00ACDC;
}

* {
```